fence/README.md

```diff
@@ -0,0 +1,153 @@
+# spec
+
+## Terminology
+
+| Term     | Definition                                                                            |
+| -------- | ------------------------------------------------------------------------------------- |
+| CO       | Container Orchestration system that communicates with plugins using CSI service RPCs. |
+| SP       | Storage Provider, the vendor of a CSI plugin implementation.                          |
+| DR       | Disaster Recovery.                                                                    |
+| RPC      | [Remote Procedure Call](https://en.wikipedia.org/wiki/Remote_procedure_call).         |
+
+## Objective
+
+Define a standard that will enable storage vendors (SP) to
+performing cluster level fencing on required addresses.
```

Comment:
`cluster level` suggests that the CSI specification also mentions clusters. It does not, so you need to clarify what a cluster is in this context. As there is no cluster object/structure in CSI, it might not be appropriate to call it cluster either. Maybe you can use CSI names for the components that you want fencing, if not, explain the components clearly.

`addresses` can be almost anything, be specific.